appinfo/routes.php

```diff
@@ -38,6 +38,7 @@
 
 		['name' => 'chattyLLM#newSession', 'url' => '/chat/new_session', 'verb' => 'PUT'],
 		['name' => 'chattyLLM#updateSessionTitle', 'url' => '/chat/update_session', 'verb' => 'PATCH'],
+		['name' => 'chattyLLM#updateSessionIsRemembered', 'url' => '/chat/update_is_remembered', 'verb' => 'PATCH'],
```

Comment:
This is a good moment to make a generic endpoint to update a session like `PATCH /chat/session/{sessionId}` which would be used to update the session title and to update is_remembered.